### PR TITLE
Export sample

### PR DESCRIPTION
**Pull request by RussTheAerialist**
Add a sample/tool for using export from the rest api.  It also supports view filters.

### CODE REVIEW

File: samples/export.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import argparse
+import getpass
+import logging
+
+import tableauserverclient as TSC
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Get all of the refresh tasks available on a server')
```
Comment (shinchris) [inline samples/export.py]: Description here and for list.py is for refresh tasks.

File: samples/export.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    logging_level = getattr(logging, args.logging_level.upper())
+    logging.basicConfig(level=logging_level)
+
+    tableau_auth = TSC.TableauAuth(args.username, password, args.site)
+    server = TSC.Server(args.server, use_server_version=True)
+    with server.auth.sign_in(tableau_auth):
+        views = filter(lambda x: x.id == args.resource_id,
+                       TSC.Pager(server.views.get))
+        view = views.pop()
+        (populate_name, option_factory_name, member_name, extension) = args.type
```
Comment (t8y8) [inline samples/export.py]: It took me a few reads to understand what's going on here. 
I think the code is fine, but some suggestions for clarity:

- Rename 'populate_name' to 'populate_func_name'
- Add a one line comment above that says something like "We've got lots of types and classes, this is code to figure out all the right functions to call" or something

File: samples/export.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        password = args.p
+
+    # Set logging level based on user input, or error by default
+    logging_level = getattr(logging, args.logging_level.upper())
+    logging.basicConfig(level=logging_level)
+
+    tableau_auth = TSC.TableauAuth(args.username, password, args.site)
+    server = TSC.Server(args.server, use_server_version=True)
+    with server.auth.sign_in(tableau_auth):
+        views = filter(lambda x: x.id == args.resource_id,
```
Comment (t8y8) [inline samples/export.py]: Code is fine, but why use a pager with filter instead of the native filtering in request options?

Can we not filter on ID yet?

Comment (RussTheAerialist) [reply samples/export.py]: I'm not sure views supports that right now.

**Pull request merged**